src/polyfills.ts

```diff
@@ -79,3 +79,7 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
  * APPLICATION IMPORTS
  */
 import 'hammerjs';
+
+
+// Add global to window, assigning the value of window itself.
+(window as any).global = window;
```

Comment:
Why is this assignment needed?

Reply:
Without this assignment, the app compiles but fails in the browser. Following is the error which pops up:
![image](https://user-images.githubusercontent.com/34459894/68060140-828ca600-fcd5-11e9-80ac-06524481b31a.png)
After research, I found this [comment](https://github.com/aws-amplify/amplify-js/issues/678#issuecomment-389106098) and it apparently fixed the issue.